pkgs/data/fonts/dotsies/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "dotsies";
+
+  regular = fetchurl {
+    name = "dotsies-regular";
+    url = "http://dotsies.org/Dotsies.ttf";
```

Comment:
We need a stable url. Otherwise it will silently break, if the author decides to upload a new version.
Since licensing is unclear ask the author, if it is ok to mirror the files somewhere else (if redistribution is allowed) to make sure they won't change. It would be also ok, if the author provides versions and keep old releases.